releases/R2024a/README.md

```diff
@@ -0,0 +1,148 @@
+| `27000`, `50115` | Required for communication between the Network License Manager and the workers. |
+| `22`, `3389` | Required for Remote Desktop functionality. This can be used for troubleshooting and debugging. |
+
+## Step 4. Create Stack
+Review or edit your stack details. You must select the acknowledgements to create IAM resources. Otherwise, the deployment produces a `Requires capabilities : [CAPABILITY_IAM]` error and fails to create resources.
+
+When you are satisfied with your stack configuration, click **Create stack**. AWS starts creating the resources for your server environment and opens you to the **Events** tab for the stack. 
+    
+>**Note**: Resource creation can take up to 20 minutes. After resource creation, it can take up to 15 minutes for the resources to be active.  
+
+## Step 5. Upload License File
+1. On the **Events** tab for your stack, wait for the status to reach **CREATE\_COMPLETE**.
+1. On the **Outputs** tab, in the row for the `MatlabProductionServerLicenseServer` key, click the corresponding URL listed under **Value** to open the Network License Manager for MATLAB Dashboard login page.
+1. Log in to the Network License Manager for MATLAB Dashboard.<ul><li>The username is **manager**.</li><li>For the password, enter the password that you entered in the **Dashboard Login** section while creating the stack.</li></ul>
+1. Upload your MATLAB Production Server license by following the instructions in the dashboard.
+
+
+## Step 6. Connect and Log In to MATLAB Production Server Dashboard
+> **Note**: The Internet Explorer web browser is not supported for interacting with the dashboard.
+
+1. On the **Outputs** tab for your stack, in the row for the `MatlabProductionServerDashboardURL` key, click the corresponding URL listed under **Value**. This URL is the HTTPS endpoint to the MATLAB Production Server Dashboard and brings you to the dashboard login page.
+1. Log in using the administrator username and password that you specified in the **Dashboard Login** section while creating the stack. For more information on how to use the dashboard, see [Manage MATLAB Production Server Using the Dashboard](https://www.mathworks.com/help/mps/server/manage-matlab-production-server-using-the-dashboard.html).  
+
+You are now ready to use MATLAB Production Server on AWS. 
+
+To run applications on MATLAB Production Server, you need to create applications using MATLAB Compiler SDK. For more information, see [Create Deployable Archive for MATLAB Production Server](https://www.mathworks.com/help/compiler_sdk/mps_dev_test/create-a-deployable-archive-for-matlab-production-server.html). 
+
+# Additional Information
+
+## Use Existing License Server in Existing VPC
+To manage MATLAB Production Server licenses, you can deploy an existing Network License Manager server for MATLAB instead of deploying a new one. To use an existing license server, on the *Deploy License Server* step of the deployment, select `No`.
```

Comment:
I think this sentence has a typo. How about : To manage MATLAB Production Server licenses, you can **use** an existing Network License Manager server for MATLAB instead of deploying a new one.